mop/management/commands/gaia_classifier.py

```diff
@@ -46,6 +47,8 @@ def handle(self, *args, **options):
                 # parameters in the EXTRA_PARAMs for each Target.  Targets with no
                 # fit parameters are ignored until they are model fitted.
                 # Fitted targets will have their class set to microlensing by default
+                print(event.extra_fields['u0'], event.extra_fields['t0'], event.extra_fields['tE'],
```

Comment:
If these output statements are to remain in operational code, please convert them to log entries.

Reply:
I just pushed a new commit to the tns_query branch with this statement removed (it was a leftover from the old version of the code).